### ISSUE TITLE
Ingest SIP EARK2 doesn't considering SIP additional vocabulary

### ISSUE THREAD

**Issue by elbre** [keeps/roda#1509]
During the ingest by the rules of the new version of the E-ARK SIP 2 is used only CSIP schema in the question of the "note type".

<xs:attribute name="**NOTETYPE**">
        <xs:simpleType>
            <xs:restriction base="xs:string">
                <xs:enumeration value="**SOFTWARE VERSION**"/>
            </xs:restriction>
        </xs:simpleType>
</xs:attribute>

But the SIP specification is allowing to use also: 

< Vocabulary Name="**VocabularyNoteType**">
        < Information>
            < p>Vocabulary derived from E-ARK< /p>
        </ Information>
        < Entry>
            < Term lang="en">**IDENTIFICATIONCODE**< /Term>
            < Definition>Describes the identification code of an agent< /Definition>
        </ Entry>
    </ Vocabulary>

Therefore ingest is failing despite the valid usage of metadata.

**Comment by hsilva-keep** [comment]
@elbre Support for E-ARK SIP version 2 is in alpha phase, so we will work on the full compliance in the next weeks.
Nevertheless, thank you for reporting and as soon as full compliance is achieved I'll close the issue (and you will be notified).

**Comment by elbre** [comment]
Good day to you guys,
I thought it will be helpful to continue in this thread instead of creating the new one.
So far I noticed the three places were Roda doesn't follow the new Eark specification.

1, missing IDENTIFICATIONCODE value as mentioned in this issue before
2, CONTENTINFORMATIONTYPE as well as the "VocabularyDetailedContentType" are using term "ERMS" instead of "SMURFERMS"
3, xmlns:csip="https://dilcis.eu/XML/METS/CSIPExtensionMETS" is now officially written with capitalised "DILCIS" (https://DILCIS.eu/XML/METS/CSIPExtensionMETS). This can sound like a minor detail but if it is written as the specification suggest the RODA will fail to ingest such package and on top of that will report the false issue: " Attribute 'OAISPACKAGETYPE' must appear on element 'metsHdr'".

**Comment by jmaferreira** [comment]
Dear @elbre ,

Thanks for your feedback. Just a question. Are you using or testing RODA at Prague city archives (AHMP)?

**Comment by elbre** [comment]
Yes, we are testing RODA in our place and consider using it as the part of our the solution. Is there any problem with that? Am I too much nosy?

**Comment by jmaferreira** [comment]
@elbre Not at all! :-) Just curious. We love to know where RODA is being implemented around the world. Thats it.

**Comment by elbre** [comment]
Glad to hear it!
I understand that this is not number one priority (not to mention current worldwide situation), but do you have at least brief idea when E-ARK2 will move out of alpha stage?

**Comment by hsilva-keep** [comment]
@elbre with some certainty, as we will re-start working on this briefly, in a month or two.

**Comment by luis100** [comment]
@elbre is this still an existing issue? I guess that as commons-ip integrated validation of specification we should have gone through this.
